state/futures/selectors.ts

```diff
@@ -110,6 +110,17 @@ export const selectMarketPrice = createSelector(
 	}
 );
 
+export const selectMarketPriceColor = createSelector(
+	selectMarketInfo,
+	(state: RootState) => state.prices,
+	(marketInfo, { offChainPriceColors }) => {
+		if (!marketInfo || !offChainPriceColors[marketInfo.asset]) return 'white';
+		const color = offChainPriceColors[marketInfo.asset]?.color;
+		const expiresAt = offChainPriceColors[marketInfo.asset]?.expiresAt;
```

Comment:
I don't think this works as expected, as if I understand correctly seems you want the color to only flash for 1 second and then expire back to white but there is no event to trigger this selector again, it will only be called on the next prices update and watching the color change intervals I'm noticing it vary quite a bit.

If we really want it to expire back to white after 1 second then we'll need some other update to trigger the change, maybe in `getPriceColors` we add a setTimeout trigger and fire another action to `resetPriceColors` or something?

Reply:
Yeah you're right - I actually had written a version with a reset before the `expiresAt` version. It also wasn't working as I expected at the time, but it may have been before I set up the selectors. I'll try that.

Reply:
I found that with a reset based on a timeout sometimes we get a new price update just before the reset. This causes the opposite problem, where the color flashes for <1 second.

Maybe we need to combine the two solutions, we need some way to trigger the selector.

Reply:
A reset which only changes any non-expired colors is probably the way. That way if we get a price update before the timeout, the reset won't change the color.